src/protocols/Protocols.h

```diff
@@ -87,6 +87,7 @@ CHIP_STANDARD_PROTOCOL(InteractionModel, 0x0001)          // Interaction Model P
 CHIP_STANDARD_PROTOCOL(BDX, 0x0002)                       // Bulk Data Exchange Protocol
 CHIP_STANDARD_PROTOCOL(UserDirectedCommissioning, 0x0003) // User Directed Commissioning Protocol
 CHIP_STANDARD_PROTOCOL(Echo, 0x0004)                      // Echo Protocol.  To be removed or standardized.
+CHIP_STANDARD_PROTOCOL(CheckIn, 0x0005)                   // CheckIn Protocol.  TODO: Check if it belongs here
```

Comment:
The Check-In protocol is contained within the secure Channel protocol. We don't need the extra entry.

Reply:
Yes, this needs a new message type added to secure channel, if we have not already done that.

Reply:
There is a msg added in secure_channel for ICD_checkin. So, removing this.